templates/vue/src/components/Lightbox/media/VideoMedia/UrlVideoMedia.vue

```diff
@@ -130,7 +130,11 @@ export default {
         if (Math.abs(currentTime - this.lastTime) > SEEK_THRESHOLD) {
           this.$emit("seeked", { currentTime })
         } else {
-          this.$emit("timeupdate", { amountViewed, currentTime })
+          var currentTimeInt = parseInt(currentTime, 10)
+          var isFiveSeconds = currentTimeInt % 5 === 0 && currentTimeInt !== 0
+          if (isFiveSeconds) {
+            this.$emit("timeupdate", { amountViewed, currentTime })
+          }
```

Comment:
@harry02031 the changes in this file seem to be unrelatetd to this PR. I believe the base branch is dealing with this already, but if the base branch is not doing what it should be doing, then these changes should be moved to the base branch where this is the objective of it, not remain here. Otherwise, these changes should be removed.